ParisBNF/abb/BNFabb195.xml

```diff
@@ -0,0 +1,683 @@
+
+                            <msItem xml:id="ms_i6.9">
+                            <locus from="120vb" to="123vb"/>
+                            <title type="complete" ref="LIT3148Zephan"/>
+                            </msItem>
+
+                            <msItem xml:id="ms_i6.10">
+                            <locus from="123vb" to="125vb"/>
+                            <title type="complete" ref="LIT3149Haggai"/>
+                            </msItem>
+
+                            <msItem xml:id="ms_i6.11">
+                            <locus from="125vb" to="136va"/>
+                            <title type="complete" ref="LIT3150Zechar"/>
+                            </msItem>
+
+                            <msItem xml:id="ms_i6.12">
+                            <locus from="136va" to="140rb"/>
+                            <title type="complete" ref="LIT3151Malachi"/>
+                            </msItem>
+
+                            <msItem xml:id="ms_i7">
+                               <locus from="141ra" to="243va"/>
+                               <title type="complete" ref="LIT2035Jerem">ኤርምያስ</title>
+                               <note>The Cycle of Jeremiah. Running title through <locus from="141r" to="243r"></locus>
+                             </note>
+
+                              <msItem xml:id="ms_i7.1">
+                            <locus from="141ra" to="226ra"/>
+                          <title type="complete" ref="LIT1685Bookof"/>
+                        <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ እግዚአብሔር፡ መሐሪ፡</hi>
+                        ወመስተሣህል፡ ቃለ፡ እግዚአብ<hi rend="rubric">ሔር፡ ዘኮነ፡ ኀበ፡ ኤርምያስ፡ ነቢይ፡</hi> ወልደ፡
+                        ቂልቅዩ፡ ዘእምውስተ፡ ካህናት፡ እለ፡ ይነብሩ፡ ውስተ፡
+                         አናቶት፡ በብሔረ፡ ብንያሚ። ወኮነ፡ ቃለ፡ እግዚአብሔር፡ ኀቤሁ፡ በመዋዕለ፡
+                             ኢዮስያስ፡ ወልደ፡ አሞጽ፡ ንጉሠ፡ ይሁዳ፡ አመ፡ ዓሠርቱ᎓ ወሠለስቱ᎓ አመተ፡ መንግሥቱ ወኮነ፡ በመዋዕለ፡ ኢዮአቂም፡
+                        </incipit>
+                      <explicit xml:lang="gez">እስከ᎓ አመ፡ ዕለተ᎓ ሞቱ፨
+                      </explicit>
+                        <note>Book of Jeremiah. Chapter numbers are written in the margins (1 to 52)
+                        are written in the margins.
+
+                        Corrections, glosses and commentaries (some in Amharic r)
+                        are written in the margins.</note>
+                          </msItem>
+
+                          <msItem xml:id="ms_i7.2">
+                          <locus from="226ra" to="228vb"/>
+                          <title type="complete" ref="LIT1202Bookof"/>
+                           <incipit xml:lang="gez"><hi rend="rubric"/>
+                           <hi rend="rubric">ዝነገረ᎓ መጽሐፍ᎓ ዘጸሐፈ᎓ ባሮክ᎓ ወልደ᎓ ኔርዩ᎓ ወልደ፡ ማሲው፡ ወልደ፡</hi>ሴደቅዮ፡ ባቢሎን፡
+                         በኃምስ፡ ዓመት፡</incipit>
+                              <explicit xml:lang="gez">በፍሥሐ᎓ ወበብርሃነ᎓ ስብሐቲሁ᎓ በምሕረቱ፡ ወበጽድቁ፨
+                          </explicit>
+                          <note>Book of Baruch.</note>
```

Comment:
I know that you like these `<note`>s with the title, but I really feel that they add no information and contribute to making the page more cluttered...

Reply:
Yes, of course, I will remove. It is mostly for me, for orientation, as the IDs frequently give no hint as to the text they denote ("Bookof"). As my mind is not digital and needs some concret points to grasp, I have to assist myself :) I will remove them at the end.

Reply:
ah, I see